cmd/openshift-install/create.go

```diff
@@ -268,13 +270,25 @@ func logComplete(directory string) error {
 	if err != nil {
 		return err
 	}
+	tfvarsFile := filepath.Join(absDir, cluster.TfVarsFileName)
+	f, err := os.Open(tfvarsFile)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+	tfVars, _ := ioutil.ReadAll(f)
+	var result map[string]interface{}
+	json.Unmarshal([]byte(tfVars), &result)
+	clusterName := result["cluster_name"]
+	baseDomain := result["base_domain"]
 	kubeconfig := filepath.Join(absDir, "auth", "kubeconfig")
 	pwFile := filepath.Join(absDir, "auth", "kubeadmin-password")
 	pw, err := ioutil.ReadFile(pwFile)
 	if err != nil {
 		return err
 	}
-	logrus.Infof("kubeadmin user password: %s", pw)
-	logrus.Infof("Install complete! The kubeconfig is located here: %s", kubeconfig)
+	logrus.Infof("Install complete! To manage the cluster with 'oc', the OpenShift CLI, first run 'export KUBECONFIG=%s'", kubeconfig)
```

Comment:
Now that we have a few lines in this footer, maybe we should split `Install complete!` and `To manage...` on separate lines?

Reply:
yes, agreed, _ahem_ only way to do that is another logrus.Infof line? : )

Reply:
Imo, I think it would be more readable.

Reply:
yes, will do